lib/kubecfg.libsonnet

```diff
@@ -63,6 +63,11 @@
   // object that conforms to a schema defined by the chart.
   parseHelmChart:: std.native('parseHelmChart'),
 
+  // validateJSONSchema(obj, schema): Validates a given object against the provided
+  // schema. Returns 'true' is the schema is valid. If this is not the case, an error stream
+  // is omitted based on the given schema's rules.
+  validateJSONSchema(obj, schema):: std.native('validateJSONSchema')(obj, schema),
```

Comment:
The function arity and the names of the formal parameters are defined in Go:

```
	vm.NativeFunction(&jsonnet.NativeFunction{
		Name:   "validateJSONSchema",
		Params: []jsonnetAst.Identifier{"obj", "schema"},
```

so we don't need a jsonnet function that calls a native function, we can just return the native function (like the other entries above)

```suggestion
  validateJSONSchema:: std.native('validateJSONSchema'),
```

The advantages are:

1. no need to duplicate the names of the formal parameters
2. avoid having a stack trace entry that points to kubecfg.libsonnet

i.e. error messages are cleaner:

```
ERROR error reading /tmp/validate.jsonnet: RUNTIME ERROR: object is invalid against the schema: jsonschema: '' does not validate with file:///Users/mkm/p/kubecfg#/additionalProperties: additionalProperties 'somethingelse' not allowed
        file:///tmp/validate.jsonnet:5:12-77    object <anonymous>
        Checking object assertions
        Field "spec"
        During manifestation
```

as opposed to

```
ERROR error reading /tmp/validate.jsonnet: RUNTIME ERROR: object is invalid against the schema: jsonschema: '' does not validate with file:///Users/mkm/p/kubecfg#/additionalProperties: additionalProperties 'somethingelse' not allowed
        internal:///kubecfg.libsonnet:69:37-82  function <anonymous>
        file:///tmp/validate.jsonnet:5:12-77    object <anonymous>
        Checking object assertions
        Field "spec"
        During manifestation
```

Reply:
Makes sense!